src/composer.json

```diff
@@ -3,15 +3,16 @@
     "description": "Wordpress Plugin for Organic Platform",
     "type": "wordpress-plugin",
     "require": {
-        "gmostafa/php-graphql-client": "^1.10",
```

Comment:
Note to self: undo insofar as possible.

Reply:
@sergunich: Nevermind, this seems to be working. I'll revert these versions. That said, ./dev up is inconsistent for me. Sometimes the command breaks somewhere, which leads to the composer not installing updates and the WP environment not being set up (hence my original difficulties accessing the database in this PR). I'll hopefully identify the issue and fix it in this sprint.